src/main/java/Decide.java

```diff
@@ -119,6 +118,28 @@ boolean evaluateLIC_8() {
         return false;
     }
 
+    boolean evaluate_LIC_9(){
+        if (params.C_PTS < 1 || params.D_PTS < 1 || params.C_PTS + params.D_PTS > params.points.length - 3
+                || params.points.length < 5)
+            return false;
+
+        for (int i = 0; i < params.points.length-2 - params.C_PTS - params.D_PTS; i++){
+            Point a = params.points[i];
+            Point b = params.points[i+1 + params.C_PTS];
+            Point c = params.points[i+2 + params.D_PTS];
+            if ((a.getX()==b.getX() && a.getY()==b.getY()) || (b.getX()==c.getX() && b.getY()==c.getY()))
```

Comment:
There's some code duplication here but I think that could be solved in a future refactor

Reply:
Yeah, an equals() function for the Point class would definitely not be a bad thing :D